client.cpp

```diff
@@ -350,8 +311,13 @@ int client::prepare(void)
     return 0;
 }
 
-void client::handle_response(struct timeval timestamp, request *request, protocol_response *response)
+void client::handle_response(unsigned int conn_id, struct timeval timestamp,
```

Comment:
@YaacovHazan I may be missing something, but what is the purpose of conn_id in handle_request/handle_response? I don't see it being used.

Reply:
I'm using it in cluter_client